prowler/lib/outputs/file_descriptors.py

```diff
@@ -77,6 +76,8 @@ def fill_file_descriptors(output_modes, output_directory, output_filename, provi
                 # FIXME: Remove this once we always use the new CSV(Output)
                 if output_mode == "csv":
                     continue
+                elif output_mode == "json-asff":
+                    continue
```

Comment:
Is this needed? As compliance does not use it, I think we can remove it

Reply:
I think is needed for now until we solve Compliance, am I right @pedrooot ?

Reply:
when all the output PRs are merged we can delete all the ifs and maintain only the specifics for compliance.

Reply:
I still think that this can be removed, this `output_mode` is not used in compliance...